EssentialCSharp.Web/Views/Shared/_Layout.cshtml

```diff
@@ -88,69 +88,76 @@
 </head>
 <body>
     <div id="app">
-        <div class="app-container">
+        <div>
             <header class="header-background">
                 <div class="banner">
                     <div class="menu-position">
                         <button class="menu-btn" v-on:click="sidebarShown = !sidebarShown">
```

Comment:
I'm wondering if we should store the sidebar state in the browser, because if I hide the toolbar currently, then go to the next page, it opens again. Which could be annoying if I've intentionally hidden it to improve my reading experience.

Reply:
https://github.com/IntelliTect/EssentialCSharpManuscript/issues/794